config/locales/views/submissions/en.yml

```diff
@@ -78,7 +81,12 @@ en:
       no_submission: You have not submitted any work yet.
       only_required_files: You may only submit the required files listed above.
       select_file: Select a file to view
+      url: URL
+      url_text: Display URL as
       version_control_warning: Submitting files here overrides any changes made using version control. Make sure you have worked on the most recent files before submitting.
     submission_summary: Submission Report
     successfully_changed: Successfully updated %{changed} results.
     unrelease_marks: Unrelease Marks
+    unsupported_url: Previews from %{host} are not supported and cannot be displayed
+    url_display_error: There was an error trying to display a preview for this URL
```

Comment:
Please change this key to `url_preview_error` to make it more consistent.

Also please change the text to "Could not display a preview for this URL. Please click on the link above to access it." (slightly more specific, and refers to alternate action)